src/v2_0_1/helpers/mod.rs

```diff
@@ -1,2 +1,5 @@
 /// validators
 pub mod validator;
+
+/// serializers
+pub mod serializer;
```

Comment:
Looks very promising. My only input is that we should  maybe call this something else? like `rfc3339_serializer` or something more telling?

Reply:
Got your point.

But my intention (understanding) was that, because its under the "helpers", this module `serializer` would be probably a collection of "serializers", if there maybe later some other needed, not only for rfc3339_serializer.

Maybe the `pub mod datetime` could/should be more like `rfc3339_serializer` ?

Reply:
Yes, I think `#[serde(with = "datetime_rfc3339")]` or something like that would be more explicit to developers.

Reply:
`rfc3339_serializer` sounds good

Reply:
Hm, it 's also a "deserializer" when I think about it so maybe `datetime_rfc3339` is better still